cpu/stm32l1/periph/i2c.c

```diff
@@ -50,25 +47,13 @@ static inline void _stop(I2C_TypeDef *dev);
  * @brief Array holding one pre-initialized mutex for each I2C device
  */
 static mutex_t locks[] =  {
-#if I2C_0_EN
-    [I2C_0] = MUTEX_INIT,
-#endif
-#if I2C_1_EN
-    [I2C_1] = MUTEX_INIT,
-#endif
-#if I2C_2_EN
-    [I2C_2] = MUTEX_INIT
-#endif
-#if I2C_3_EN
-    [I2C_3] = MUTEX_INIT
-#endif
+    [0] = MUTEX_INIT,
+    [1] = MUTEX_INIT,
```

Comment:
@thomaseichinger Hmm, I'm not sure how to implement that.
If I try that, I only get errors.